modules/js/src/embindgen.py

```diff
@@ -142,6 +142,21 @@
               'BFMatcher': ['isMaskSupported', 'create'],
               '': ['FAST', 'AGAST', 'drawKeypoints', 'drawMatches']}
 
+photo = {'': ['createAlignMTB', 'createCalibrateDebevec', 'createCalibrateRobertson',
+              'createMergeDebevec', 'createMergeMertens', 'createMergeRobertson',
+              'createTonemapDrago',
+              'createTonemapMantiuk', 'createTonemapReinhard'],
+        'AlignMTB' : ['calculateShift', 'shiftMat', 'computeBitmaps', 'getMaxBits', 'setMaxBits', 'getExcludeRange', 'setExcludeRange', 'getCut', 'setCut'],
+        'CalibrateDebevec' : ['getLambda', 'setLambda', 'getSamples', 'setSamples', 'getRandom', 'setRandom'],
+        'CalibrateRobertson' : ['getMaxIter', 'setMaxIter', 'getThreshold', 'setThreshold', 'getRadiance'],
+        'MergeDebevec' : ['process'],
+        'MergeMertens' : ['process', 'getContrastWeight', 'setContrastWeight', 'getSaturationWeight', 'setSaturationWeight', 'getExposureWeight', 'setExposureWeight'],
+        'MergeRobertson' : ['process'],
+        'TonemapDrago' : ['getSaturation', 'setSaturation', 'getBias', 'setBias'],
+        'getSigmaColor', 'setSigmaColor', 'getSigmaSpace', 'setSigmaSpace'],
```

Comment:
this line fails the build, could you please double check?

Reply:
thanks, I fixed that now. I tried with commenting out everything apart from `createCalibrateDebevec`., just to check exposing of 1 single function. Still not exposed.